models/tmp/stg_stripe__pricing_tmp.sql

```diff
@@ -0,0 +1,40 @@
+{{ config(enabled=var('stripe__using_subscriptions', True)) }}
```

Comment:
Shouldn't the `{% if var('stripe__price', does_table_exist('price')) %}` logic be in this model? If not, the model would fail at this step if the customer doesn't have the price table.

Reply:
Good catch, updated!

Reply:
Good catch, updated!